config/routes.rb

```diff
@@ -66,6 +63,8 @@
                                               destroy], path: "listings/categories"
 
       resources :comments, only: [:index]
+      resources :data_update_scripts, only: %i[index show]
+      post "/data_update_scripts/:id/force_run", to: "data_update_scripts#force_run"
```

Comment:
this can be Rails-yfied with:

```suggestion
      resources :data_update_scripts, only: %i[index show] do
        member do
          post :force_run
        end
      end
```

this way you also get the proper path helper:

```ruby
[8] pry(main)> show-routes -G force_run
                force_run_admin_data_update_script POST     (/locale/:locale)/admin/data_update_scripts/:id/force_run(.:format)                    admin/data_update_scripts#force_run {:locale=>nil}
```